app/experimenter/jetstream/models.py

```diff
@@ -78,6 +78,12 @@ class JetstreamDataPoint(BaseModel):
 
 
 class JetstreamData(BaseModel):
+    """
+    Parameters:
+        __root__: List[JetstreamDataPoint] = []
+            The list should be filtered as needed coming in (e.g., by a given segment).
+    """
```

Comment:
Do you need this if the property below is already type annotated?  Or just to add the description?

Reply:
The main thing I wanted to get across is that the list in `__root__` should already be filtered at instantiation time. Previously we were doing some filtering to make sure the data points were `Segment.ALL`, but I removed that and placed the responsibility on the code creating the object. Happy to discuss a better way to accomplish that goal though.